### ISSUE TITLE
[Screen reader - BOT Framework - Web Chat > Enter Keyword]: 'Adaptive card' group label is not associated with the first/ last elements below.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4722]

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev3;#HCL;#MAS1.3.1;#Accessibility;#Benchmark;#Win11-Edge(Chromium);#BotFrameworkWebChat-Apr23;
